### ISSUE TITLE
Potential compatibility with create to prevent item loss/dupes

### ISSUE THREAD

**Issue by Hophenby** [P3pp3rF1y/SophisticatedStorage#340]
to reproduce this bug:
step 1 apply compression upgrade to the container and add some item like this:
![image](https://github.com/P3pp3rF1y/SophisticatedStorage/assets/66473090/7b5312ad-047f-44e3-9f70-2ea92f9adc79)

step 2 attach this container to a Create contraption
![image](https://github.com/P3pp3rF1y/SophisticatedStorage/assets/66473090/c6d952ad-059e-41d8-b854-74a8ecd7395a)

step 3 open the barrel on the contraption and take the item inside
![image](https://github.com/P3pp3rF1y/SophisticatedStorage/assets/66473090/f38e2023-b66e-4139-9778-66d941ed2f6c)

step 4 disassemble the contraption and repeat the steps above

Create mod version 0.5.1e
Sophisticated Storage version 0.8.51.605

**Comment by P3pp3rF1y** [comment]
Yup, sure that will require a proper compatibility for create which is not possible at the moment. Not really a bug I can fix on my side because create in general makes assumptions about storages it's connecting and so it assumes a dummy storage and just copies the stacks over. Without compatibility support and proper compatibility implemented the only "fix" at the moment is banning sophisticated storage from being attachable to create using create configs if such thing exists. 

I would like to see if create team is open to adding compatibility support so that I could add compatibility on my side. Thus I will leave this open for possible future compat.

**Comment by P3pp3rF1y** [comment]
We now have a proper full featured Sophisticated Storage Create compat in https://www.curseforge.com/minecraft/mc-mods/sophisticated-storage-create-integration for 1.20 and 1.21 so I am closing this
